data-plane/core/src/main/java/dev/knative/eventing/kafka/broker/core/oidc/TokenVerifierImpl.java

```diff
@@ -56,7 +56,7 @@ public Future<JwtClaims> verify(String token, String expectedAudience) {
             } catch (InvalidJwtException e) {
                 promise.fail(e);
             }
-        });
+        },false);
```

Comment:
Did you run java code formatting https://github.com/knative-extensions/eventing-kafka-broker/blob/main/DEVELOPMENT.md#java ?

Reply:
Luckily not before you told about it, Now I run it & got success. 
![Screenshot from 2024-02-26 22-11-10](https://github.com/knative-extensions/eventing-kafka-broker/assets/112557191/7cfea197-2029-4347-a84d-f6f2eeb0917b)

Another thing I want to bring your attention on, somewhere I found `executeBlocking(handler<promise<T>>, ordered)` is depricated. [source](https://vertx.io/docs/apidocs/io/vertx/core/Vertx.html). 
Happy learning :)

Reply:
Hi @parth721,
yes you need to push the changes after you run `./hack/update-codegen.sh`